lib/dap/input.rb

```diff
@@ -50,17 +51,23 @@ def read_record
   # JSON Input (line-delimited records)
   #
   class InputJSON
-    
+
     include FileSource
 
     def initialize(args)
       self.open(args.first)
     end
 
-    def read_record  
+    def read_record
       line = self.fd.readline rescue nil
       return Error::EOF unless line
-      json = Oj.load(line.strip) rescue nil
+      begin
+        json = Oj.load(line.strip)
+      rescue
+        $stderr.puts "\nRecord is not valid JSON and will be skipped."
```

Comment:
This output, when handed large results, creates a wall of text.  Without the extra line break before there would be no visual breaks at all.  I've tossed you a screenshot sample.